sdk/synapse/synapse-artifacts/review/synapse-artifacts.api.md

```diff
@@ -241,7 +239,7 @@ export interface AmazonS3Location extends DatasetLocation {
 // @public
 export interface AmazonS3ReadSettings extends StoreReadSettings {
     deleteFilesAfterCompletion?: any;
-    enablePartitionDiscovery?: boolean;
+    enablePartitionDiscovery?: any;
```

Comment:
Please request for api-view review
https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/RLC-quickstart.md#before-you-start
![image](https://github.com/Azure/azure-sdk-for-js/assets/9943211/4eebe8cb-05bb-45a0-9c73-2eb9e19a9f51)